bundles/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/message/CMessage.java

```diff
@@ -26,6 +26,7 @@
 
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.net.util.Base64;
```

Comment:
Can you please check for usage of `org.apache.commons.net.util.Base64` in the tests too?

Reply:
This is because my note: decodebase64 still pending as output results differ from util.base64 (apache decoder is more lenient)

Spend all ready quite some time on this, but I have not yet found a way to have regular decoder accept all strings.

Hence this pr only has the encoding instances replaced.

Reply:
Note.. I have also read into thre RFC for base64. The RFC is bit ' sloppy' wrt how the ending is defined. The language is ' may add ' some specific ending. 
 seems the decoding in java.util.base64 implementation is not allowing this specific ending, implementing it in a strict way and gives an error.

Reply:
darn... wasted 2 evenings trying to understand the differences in the implementation, to now find out the testcode in TCommandTest.java was having a bug.

Reply:
I am glad you found the bug and with it a solution for `org.apache.commons` replacement.